x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/create.ts

```diff
@@ -46,11 +46,13 @@ export const registerSiemRuleMigrationsCreateRoute = (
             const ctx = await context.resolve(['securitySolution']);
             const ruleMigrationsClient = ctx.securitySolution.getSiemRuleMigrationsClient();
 
+            const migrationId = uuidV4();
```

Comment:
This is done at line `44` now, must be a conflict leftover

Reply:
Yeah I noted down earlier that the merge conflicts I resolved needs a second look, il try to fix that up in the weekend. If there is anything outside the line messed up by the merge conflict let me know :1+

Reply:
That's all the rest looks good 👍